jaxpm/painting.py

```diff
@@ -6,10 +6,18 @@
 
 
 def cic_paint(mesh, positions, weight=None):
```

Comment:
I think it is better to rollback changes on painting.py
Because in my PR painting function are different.
I will add the numpy doc later